registry/requirements.txt

```diff
@@ -22,7 +22,7 @@ MarkupSafe==1.0
 mixpanel==4.3.2
 oauthlib==2.0.2
 packaging==16.8
-PyMySQL==0.7.11
+psycopg2==2.7.3.2
```

Comment:
it'll be interesting to see if we can lock onto this specific version...

Reply:
Not sure what you mean? This does lock it to this version, right?